frame/composable-traits/src/clearing_house.rs

```diff
@@ -38,13 +45,40 @@ pub trait ClearingHouse {
 	/// ## Returns
 	/// The new market's id, if successful
 	fn create_market(config: &Self::MarketConfig) -> Result<Self::MarketId, DispatchError>;
+
+	/// Open a position in a market
+	///
+	/// This may result in the following outcomes:
+	/// * Creation of a whole new position in the market, if one didn't already exist
+	/// * Increase in the size of an existing position, if the trade's direction matches the existion position's one
```

Comment:
"matches the _existion_ position's one", wouldn't it be '_existing_'?

Reply:
Good catch, that's a typo